mandatory/3-function-output.js

```diff
@@ -1,14 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
+  // generates random floating numbers between zero and 10 but never 10
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function s(w1, w2) {
+  // stringing together two strings-no spaces- and returns a string
```

Comment:
A better way of commenting is: join two strings with no spaces

Reply:
Okay thanks